pg_search/src/postgres/customscan/pdbscan/exec_methods/top_n.rs

```diff
@@ -232,10 +228,11 @@ impl ExecMethod for TopNScanExecState {
             }
 
             // set the chunk size to the scaling factor times the limit
-            // on subsequent retries, multiply the chunk size by `RETRY_SCALE_FACTOR`
-            self.chunk_size = (self.chunk_size * RETRY_SCALE_FACTOR)
+            // on subsequent retries, multiply the chunk size by the scale factor
+            // but do not exceed the max chunk size
+            self.chunk_size = ((self.chunk_size as f64 * self.scale_factor) as usize)
```

Comment:
Re-reading this, I'm on the fence about one setting vs two... sorry.

For one thing: it feels like the defaults here were already ~decent... and so the effect of removing the separate setting for `RETRY_SCALE_FACTOR` is that the default value gets worse.

I don't feel strongly about it either way, but if you think that two settings would be better, then I'll take back what I said about merging them.

Reply:
...having said that, we're already starting by scaling the value by the dead fraction, so maybe one setting is plenty.

Reply:
Actually, let's leave it as one setting. The dead fraction is already supposed to be capturing this: if it isn't, then one setting is sufficient.